docs/audits/gpt-errors-overall.md

```diff
@@ -0,0 +1,16 @@
+# Minimize GPT API related errors
+
+## Overview
+
+This audit checks for the presence of warning, errors, and other exception related messages coming from GPT. Reducing errors
+can help ensure that user experience is consistent and working as intended.
+
+## Recommendations
+
+Observe the details of the audit to see more specific details on the errors affecting the page. Run the [Publisher Ads Audits for Lighthouse](https://developers.google.com/publisher-ads-audits)
+regularly when making changes to the webpage to minimize the introduction of possible errors.
+
+## More information
```

Comment:
Same as the other guide, can we put some details about how this audit works here?

Reply:
Done, also moved the guide paragraph up to the recommendations section.